mfhdf/nctest/testcdf.h

```diff
@@ -54,9 +54,7 @@ struct netcdf {
     struct cdfatt atts[MAX_TEST_ATTS]; /* attributes */
 };
 
-extern struct netcdf test; /*
-                            * in-memory netcdf structure, kept in sync
-                            * with disk netcdf
-                            */
+/* In-memory netcdf structure, kept in sync with disk netcdf */
+extern struct netcdf *test_g;
```

Comment:
The changes to nctest are extensive, but it's really just renaming test to test_g and making it dynamic, so &test --> test_g and test.foo --> test_g-->foo.